js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -383,13 +383,41 @@ const AssetGraphExplorerWithData = ({
     selectNodeById(e, nextId);
   };
 
+  const selectAllGroupNodesById = React.useCallback(
+    (e: React.MouseEvent<any> | React.KeyboardEvent<any>, groupId: string) => {
+      const assets = groupedAssets[groupId] || [];
+      const childNodeTokens = assets.map((n) => tokenForAssetKey(n.assetKey));
+
+      const existing = explorerPath.opNames[0]!.split(',');
+
+      const nextOpsNameSelection = childNodeTokens.every((token) => existing.includes(token))
+        ? uniq(without(existing, ...childNodeTokens)).join(',')
+        : uniq([...existing, ...childNodeTokens]).join(',');
+
+      onChangeExplorerPath(
+        {
+          ...explorerPath,
+          opNames: [nextOpsNameSelection],
+        },
+        'replace',
+      );
+    },
+    [groupedAssets, explorerPath, onChangeExplorerPath],
+  );
+
   const selectNodeById = React.useCallback(
     (e: React.MouseEvent<any> | React.KeyboardEvent<any>, nodeId?: string) => {
       if (!nodeId) {
         return;
       }
-      if (isGroupId(nodeId)) {
-        zoomToGroup(nodeId);
+      const modifiedNodeId = '__repository__@' + nodeId;
```

Comment:
for some reason when running dagster dev locally, the node ids for asset groups are missing the `__repository__@` prefix (they're just in the form `repo:groupname`)

Reply:
Huh, I haven't seen this before, maybe @bengotow  knows?

Reply:
`dagster dev -f ~/Documents/my_test_file.py` is how I've been running it - maybe bc it's not a python module? not sure

Reply:
So does selecting group nodes not work at all then?

Reply:
As far as I can tell, yes. It doesn't properly zoom to the group when I select it.